File: C#/DSA/ArrayStack.cs
using System;
using System.Linq;

namespace Algorithms
{
    public class ArrayStack<T>
    {
        private T[] array = new T[10];
        private int count = 0;

        public ArrayStack()
        {
        }
        public void Push(T item)
        {
            array[count] = item;
            count++;
        }

        public T Pop()
        {
            var item = array[--count];
            return item;
        }

        public override string ToString()
        {
            var str = "";
            array.ToList().ForEach(x => str = str + x.ToString() + " ");
            return str;
        }
    }
}

File: Javascript/array-stack.js
class ArrayStack {
    constructor() {
        this.array = [];
        this.count = 0;
    }

    push(item) {
        this.array[this.count] = item;
        this.count++;
    }
    pop() {
        return this.array[--this.count];
    }
    display() {
        console.log(this.array);
    }
}
module.exports = ArrayStack;
